app/graphql/types/query_type.rb

```diff
@@ -37,8 +37,7 @@ def events(after: nil, type: nil)
   end
 
   def employee_events
-    # dummy return value for now because cannot return null
-    # will be filled in with database calls in a later PR
+    Clients::BamboohrClient.find_employees.update_employees
```

Comment:
Should we be saving employees to the database when we query for employee events in graphql? It seems like it will still have a long request time so we probably want to save them asynchronously via sidekiq.